person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
```

Comment:
- [ ] Kindly implement getter for the id variable as stated in the requirements: 
![image](https://user-images.githubusercontent.com/91486791/218545722-83f1d0f4-0ad8-4ea8-85c0-cfdd53e37cd0.png)